packages/docs/src/components/promoted/Script.vue

```diff
@@ -4,7 +4,7 @@
 
 <script lang="ts">
   // Utilities
-  import { wait } from '@/util/helpers'
+  // import { wait } from '@/util/helpers'
```

Comment:
dev code?

Reply:
It breaks the ad if it takes more than 1000ms to load, not sure what it's actually for so I left it